src/shared/views/BuildingFootprint/FootprintDisplay.js

```diff
@@ -100,7 +210,7 @@ export default class FootprintDisplay extends React.Component<Props> {
         return [sw, nw, ne, se];
     };
 
-    getTileUrl = (tile: Tile) => {
+    getTileUrl = (tile: Tile): string => {
```

Comment:
just as a general question: this means that we can only rely on `Bing` if we do it with the TMS tiles?

Reply:
for now, yes. I think we should refactor the code to have a small imagery url interpolation library shared across all project types. https://github.com/mapswipe/mapswipe/issues/433